svc/FetchService.ts

```diff
@@ -72,6 +77,22 @@ export class FetchService extends HoistService {
         }
     }
 
+    /**
+     * Set the token (header name) to be used for correlationId tracking.  Set false to disable
```

Comment:
Rename to `setCorrelationIdHeaderName` ?

Reply:
Lee had the same thought.  Changed to `setCorrelationIdHeaderKey`. Does that sound ok to you, or would you prefer `name > key` ?

Reply:
Great, thanks - it was mostly the word "token" that seemed too ambiguous as to whether we were talking about a key or a value.  I think with headers "name" is the common parlance for what we're talking about, but the net result is that this string is used as a key in an object we pass to the fetch API, so either term sounds fine and clear enough to me. 👍